doc/source/ray-core/tips-for-first-time.rst

```diff
@@ -4,7 +4,8 @@ Tips for first-time users
 Ray provides a highly flexible, yet minimalist and easy to use API.
 On this page, we describe several tips that can help first-time Ray users to avoid some
 common mistakes that can significantly hurt the performance of their programs.
-For an in-depth treatment of advanced design patterns, please refer to our
+For an in-depth treatment of advanced design patterns, please refer to respective :ref:`tasks <task-patterns>`
+and :ref:`actors <actor-patterns>` design patterns page.
```

Comment:
With the new core doc structure, we combined task and actor patterns into top level design patterns section.

Reply:
Yes, I know, hence I added these as references to check design patterns. The original sentence was incomplete. It read as " For an in-depth treatment of advanced design patterns, please refer to our". I just added links to the respective design patterns which made more sense.

Reply:
Oh, I'm saying you can use reference `<core-patterns>` since `<task-patterns>` and `<actor-patterns>` will be removed.

Reply:
Got it! Will update to ref core-patterns once I get into the office.